drivers/flash/flash_stm32_qspi.c

```diff
@@ -1108,8 +1107,6 @@ static int flash_stm32_qspi_init(const struct device *dev)
 	}
 #endif /* CONFIG_FLASH_PAGE_LAYOUT */
 
-	LOG_INF("Device %s initialized", dev->name);
-
```

Comment:
any reason why the LOG_INF removal ?

Reply:
Not present in the majority of the drivers. Maybe this can be done as `DBG` if required.